azure-stack/hci/upgrade/install-enable-network-atc.md

```diff
@@ -227,27 +223,30 @@ ProvisioningStatus          : Completed
 
 Ensure that each intent added has an entry for the host you're working on. Also, make sure the **ConfigurationStatus** shows **Success**.
 
-If the **ConfigurationStatus** shows **Failed**, check to see if the error message indicates the reason for the failure. For some examples of failure resolutions, see [Common Error Messages](../deploy/network-atc.md#common-error-messages).
+If the **ConfigurationStatus** shows **Failed**, check to see if the error message indicates the reason for the failure. You can also review the Microsoft-Windows-Networking-NetworkATC/Admin event logs for more details on the reason for the failure. For some examples of failure resolutions, see [Common Error Messages](../deploy/network-atc.md#common-error-messages).
 
 ### Step 8: Rename the VMSwitch on other nodes
 
 In this step, you move from the node deployed with Network ATC to the next node and migrate the VMs from this second node. You must verify that the second node has the same `VMSwitch` name as the node deployed with Network ATC.
 
-This is a non-disruptive change and can be done on all the nodes simultaneously. Run the following command:
+> [!IMPORTANT]
+> After the virtual switch is renamed, you must disconnect and reconnect each virtual machine so that it can appropriately cache the new name of the virtual switch. This is a disruptive action that requires planning to complete. If you do not perform this action, live migrations will fail with an error indicating the virtual switch doesn't exist on the destination.
+
+Renaming the virtual switch is a non-disruptive change and can be done on all the nodes simultaneously. Run the following command:
 
 ```powershell
 #Run on the node where you configured Network ATC
-Get-VMSwitch | ft name
+Get-VMSwitch | ft Name
 
 #Run on the next node to rename the virtual switch
 Rename-VMSwitch -Name 'ExistingName' -NewName 'NewATCName'
 ```
 
-After your switch is renamed, disconnect and reconnect your vNICs for the `VMSwitch` name change to go through. Once the change goes through, on each node, run the following commands:
+After your switch is renamed, disconnect and reconnect your vNICs for the `VMSwitch` name change to go through. The command below can be used to perform this action for all VMs:
```

Comment:
@helmste - I think it's important that customers know to wait on the change to go through before they run the next set of commands. We can adjust this to: **Once the name change goes through, run the following commands on each node.**

Reply:
@ronmiab - at this point, the virtual switch has already been renamed.  That change has already gone through in the previous set of commands.  The next step is to run the commands it is referring to in the next lines so each VM will pick up the new switch name.